mdb-realm/cleanup/action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   privateapiKey:
     required: true
     description: The private API key.
+  clusterName:
+    required: true
```

Comment:
This should not be required - if it's not provided, we generate one.

Reply:
Yes, I saw that. But deleting a cluster with a newly generated name doesn't make sense to me. It allways fails. That's why I decided to be required. We have also DeleteAllCluster that could be used for all clusters.

Reply:
If you don't provide a name for the deploy action, we generate a deterministic name. In the same workflow, if you call the `cleanup` action, the generated name will be the same as the name we generate in the deploy action. So the cluster will exist - that's how we use those in the .NET workflow.

Reply:
I see. Ok I will make it not required.

Reply:
Done.